contracts/instruments/RibbonThetaVault.sol

```diff
@@ -262,22 +279,29 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @param share is the number of vault shares to be burned
      */
     function withdraw(uint256 share) external nonReentrant {
-        uint256 withdrawAmount = _withdraw(share, false);
+        uint256 withdrawAmount = _withdraw(share, false, false);
         IERC20(asset).safeTransfer(msg.sender, withdrawAmount);
     }
 
     /**
      * @notice Burns vault shares and checks if eligible for withdrawal
      * @param share is the number of vault shares to be burned
      * @param isScheduled is whether the withdraw was scheduled
+     * @param isFeeless is whether or not to charge the regular `instantWithdrawFee`
      */
-    function _withdraw(uint256 share, bool isScheduled)
-        private
-        returns (uint256)
-    {
+    function _withdraw(
+        uint256 share,
+        bool isScheduled,
+        bool isFeeless
+    ) private returns (uint256) {
         (uint256 amountAfterFee, uint256 feeAmount) =
             withdrawAmountWithShares(share);
 
+        if (isFeeless) {
+            amountAfterFee = amountAfterFee.add(feeAmount);
+            feeAmount = 0;
+        }
```

Comment:
We can avoid writing these checks by setting the withdrawal fee to 0, see the [setWithdrawalFee function](https://github.com/ribbon-finance/structured-products/blob/578fbf13a74ceb616c035335a30755b44f3678a6/contracts/instruments/RibbonThetaVault.sol#L197)

Reply:
do we want to allow feeless withdraws if they aren't migrating to the V2 vault?

Reply:
Yea, I don't really see a problem with doing so. Helps simplify the migration.